scripts/External Integration Clustering/case2_clustering.py

```diff
@@ -0,0 +1,36 @@
+from relevanceai import Client
```

Comment:
Add quick comments at the top to describe what is going to happen in this file please :)